### PR TITLE
Add skeleton project to docs

### PR DESCRIPTION
**Pull request by vladimirkosmala**

### CODE REVIEW

File: Skeleton/index.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
+// Import the backend framework
+const {HttpServer, Log, WebSocket, WebSocketMessage} = require('Gui');
+
+// Activate debug logs
+Log.configure({winston: {}});
```
Comment (awegrzyn) [inline Skeleton/index.js]: I'm not sure this line is required

File: Skeleton/package.json [hunk 2]
```diff
@@ -0,0 +1,14 @@
+{
```
Comment (awegrzyn) [inline Skeleton/package.json]: Having another package.json within the project won't break anything?

File: Skeleton/index.js (hunk 1 above)
Comment (vladimirkosmala) [reply Skeleton/index.js]: Indeed, a default is created on the fly

File: Skeleton/package.json (hunk 2 above)
Comment (vladimirkosmala) [reply Skeleton/package.json]: it's ok to have a package.json on each folder you want to have special dependencies.
the resolver only search for node_modules by going up in the file tree

**Comment by vladimirkosmala** [comment]
@awegrzyn backend without OAuth should send token and userid like usual for the front to work well. Let's say a random token and "Anonymous" userid. Then the front will work again in this mode.
The front now throw when this occurs.

**Comment by awegrzyn** [comment]
@vladimirkosmala Indeed very good remark, I'll implement changes as you described

**Comment by awegrzyn** [comment]
Okay, it's been addressed in https://github.com/AliceO2Group/Gui/pull/78

**Pull request merged**